cmd/log.go

```diff
@@ -4,21 +4,33 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 	"time"
 
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+
 	log "github.com/sirupsen/logrus"
 )
 
-func init() {
+// initializeLoggerForConfig will configure the logger so that it matches the
+// user's specified configuration, such as configuring output files.
+func initializeLoggerForConfig(config runtime.Config) error {
 	time := time.Now().Unix()
-	logname := fmt.Sprintf("preflight-%d.log", time)
+	logname := strings.Join([]string{
+		config.CLILogFile, fmt.Sprint(time)}, "-")
```

Comment:
What do you think about having just 1 log file that gets rewritten with each run of preflight?  I'm not sure users need all the log files.  I think they will just need the log file from the last run of the tool. They can always copy it somewhere else manually if they want to hold on to it.

Reply:
That works for me. If someone wants to have timestamps, they can always pass a preflight-$(date).log as env/cli param.

Reply:
I like this. I'll work on it tomorrow.

Reply:
This is done - we no longer have timestamped log files, and instead just write wherever the user requests, or `preflight.log` if no user input was provided, truncating if the file already exists with data.